test/support/fixtures/products_fixtures.ex

```diff
@@ -0,0 +1,23 @@
+defmodule WineOInventoryApi.ProductsFixtures do
+  @moduledoc """
+  This module defines test helpers for creating
+  entities via the `WineOInventoryApi.Products` context.
+  """
+
+  @doc """
+  Generate a wine.
+  """
+  def wine_fixture(attrs \\ %{}) do
+    {:ok, wine} =
+      attrs
+      |> Enum.into(%{
+        description: "some description",
+        name: "some name",
+        quantity: 42,
+        rating: 5
+      })
+      |> WineOInventoryApi.Products.create_wine()
```

Comment:
Be careful with fixtures it's east to become overly reliant on them. for the tests like the ones you've written so far, I'd hold off on creating a fixture until I really see the need for it.

Reply:
This fixture was provided to me when I ran the context generator test. It came in handy in the `wine_view_test.exs` file.